packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -162,10 +162,6 @@ function createDeliverTxResponseErrorMessage(result: DeliverTxResponse): string
   return `Error when broadcasting tx ${result.transactionHash} at height ${result.height}. Code: ${result.code}; Raw log: ${result.rawLog}`;
 }
 
-function createDefaultRegistry(): Registry {
-  return new Registry([...defaultStargateTypes, ...wasmTypes]);
-}
-
```

Comment:
Yeah, removing them makes more sense. No need to keep functions for things that are not reused.